webpack.config.js

```diff
@@ -23,7 +23,7 @@ const getPlugins = () => {
         isProduction ? 'production' : 'development'
       ),
       'process.env.API_URL': JSON.stringify(
-        process.env.API_URL || 'http://localhost:8085/api'
+        process.env.API_URL || 'http://localhost:3000/api'
```

Comment:
No me andaba, encontré aquí un problema. Para  poder hacer que  el front se  comunique con el BE tuve que que quitar el /api  de la API_URL